site/docs/v1/tech/release-notes.adoc

```diff
@@ -38,7 +38,7 @@ include::docs/v1/tech/__database-migration-warning.adoc[]
 === Fixed
```

Comment:
Just for reference, we need to ensure all items in this milestone ( https://github.com/FusionAuth/fusionauth-issues/milestone/83 ) are covered here. It may be.. already, but you can review for coverage.